cmd/statediff/main.go

```diff
@@ -0,0 +1,294 @@
+package main
```

Comment:
We have a folder for tools (./tools). Do you think it could be under this folder and be its own go mod? (Package name cosmossdk.io/tools/statediff)

Reply:
We are going to put this code in a fresh repository per https://github.com/orijtech/statediff work on it in iterations, just send over a PR with the .github/workflows/statediff.yml and then move that repo under github.com/cosmos/statediff otherwise there will be so much noise under this repo.

Reply:
what is the reasoning to have it be its own repo? not saying anything bad about it but mainly worried about maintain of the tool.

Reply:
@tac0turtle there is so much noise and we cannot experiment and move fast and effectively if we have it reside in this massive centralized repo for which flakes will cause lots of PR delays. Once we have it under cosmos/statediff and stable we can move it to the cosmos-sdk but really right now it'll just be noisy to try to implement in the cosmos-sdk.

Reply:
yes that is fine, was mainly curious when the tool is done.

Reply:
Once it is all ready, we shall move it appropriately def. Even much easier to bundle it in here. Thanks for the good question!